.github/workflows/ci.yml

```
@@ -0,0 +1,30 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Link Checker
+      id: lc
+      uses: peter-evans/link-checker@v1
+      with:
+        args: -v -r docs
+    - name: Fail if there were link errors
+      run: exit ${{ steps.lc.outputs.exit_code }}
+
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: git config --global user.name ${{ secrets.GH_NAME }}
+      - run: git config --global user.email ${{ secrets.GH_EMAIL }}
+      - run: echo "machine github.com login ${{ secrets.GH_NAME }} password ${{ secrets.GH_TOKEN }}" > ~/.netrc
+      - run: cd website && yarn install && GIT_USER="${{ secrets.GH_NAME }}" yarn run publish-gh-pages
```

suggest change it to the following so it's consistent with https://github.com/DataHighway-DHX/documentation/blob/master/MAINTENANCE.md#publishing-live-updates-to-github-pages
```suggestion
      - run: cd website && yarn install && GIT_USER="${{ secrets.GH_NAME }}" CURRENT_BRANCH=master USE_SSH=true yarn run publish-gh-pages

```